tests/unit/pywbemcli/simple_interop_mock_script.py

```diff
@@ -14,8 +14,10 @@
 import six
 import pywbem_mock
 
-# interop namespace used by this mock environment
+# Namespaces used by this mock environment.
+# Must be consistent with namespace pragmas in main MOF file.
```

Comment:
Suggestion: We might explain a bit and also point back to the issue maybe since are stepping in some territory that can be confusing.

Reply:
DONE - changed to:
```
# Namespaces set up by this mock environment.
# Must be consistent with the namespace pragmas in the main MOF file, because
# the pywbem MOF compiler does not create any namespaces.
```